src/main/java/com/arpnetworking/metrics/StopWatch.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2017 Inscope Metrics, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.metrics;
+
+import com.arpnetworking.metrics.impl.TsdStopWatch;
+
+/**
+ * Interface for a thread safe stop watch. The stop watch starts on creation
+ * and must be stopped to access the elapsed time. Once stopped the instance
+ * must be discarded and cannot be started again. The design is minimal to
+ * reduce contention at the cost of object creation because timers in this
+ * library represent samples and do not reuse their underlying stop watches.
+ * However, it is possible to build a stateful timer around this but you
+ * will need to add stricter synchronization or else give up thread safety.
+ *
+ * @author Ville Koskela (ville dot koskela at inscopemetrics dot com)
+ */
+public interface StopWatch {
+
+    /**
+     * Create and start a new {@link StopWatch} instance.
+     *
+     * @return new started {@link StopWatch} instance.
+     */
+    static StopWatch start() {
```

Comment:
This is about the only thing I'm concerned about.  Why have the static on the interface?